### ISSUE TITLE
JSONField.bound_data raises TypeError when data is None

### ISSUE THREAD

**Issue by strokirk** [laymonage/django-jsonfield-backport#32]
Hi,

When a bound form with no value provided for the `JSONField` is rendered, it will currently crash.

```python
class JSONForm(Form):
    json_field = JSONField(required=False)
form = JSONForm({})
assert form.as_p()
```

raises

```
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
```

A fix has been created already by @AlexHill here: https://github.com/django/django/pull/13844

**Comment by laymonage** [comment]
Thanks for the report. That patch hasn't been merged yet, so I don't have an ETA for this.

As a side note, I would retire this project as soon as support for Django 2.2 ends.

**Comment by laymonage** [comment]
Fix released in 1.0.4 :tada:
